inginious/frontend/environment_types/kata.py

```diff
@@ -0,0 +1,50 @@
+from inginious.frontend.environment_types.env_type import FrontendEnvType
+
+
+class KataEnvType(FrontendEnvType):
+    @property
+    def id(self):
+        return "kata"
+
+    @property
+    def name(self):
+        return _("Kata container")
+
+    def check_task_environment_parameters(self, data):
+        out = {}
+        # Ensure that run_cmd is in the correct format
+        if data.get('run_cmd', '') == '':
+            out['run_cmd'] = None
+        else:
+            out['run_cmd'] = data['run_cmd']
+
+        # Response is HTML
+        out["response_is_html"] = data.get("response_is_html", False)
+
+        # Network access in grading container?
+        out["network_grading"] = data.get("network_grading", False)
+
+        # Limits
+        limits = {"time": 20, "memory": 1024, "disk": 1024}
+        if "limits" in data:
+            try:
+                limits['time'] = int(data["limits"].get("time", 20))
+                hard_time = data["limits"].get("hard_time", '')
+                if hard_time.strip() == '':
+                    hard_time = 3*limits['time']
+                else:
+                    hard_time = int(hard_time)
+                limits['hard_time'] = hard_time
+                limits['memory'] = int(data["limits"].get("memory", 1024))
+
+                if limits['time'] <= 0 or limits['hard_time'] <= 0 or limits['memory'] <= 0 or \
+                        limits['disk'] <= 0:
+                    raise Exception("Invalid limit")
+            except:
+                raise Exception("Invalid limit")
+        out["limits"] = limits
+
+        return out
+
+    def studio_env_template(self, templator, task, allow_html: bool):
+        return templator.course_admin.edit_tabs.env_docker(task.get("environment_parameters", {}), allow_html)
```

Comment:
Here you call the ``env_docker`` template but you define a ``env_kata`` below... ?? If it's the same template, maybe it would be more clear to call it ``env_container`` or something like that.